lib/components/table.vue

```diff
@@ -17,7 +17,8 @@
                     :tabindex="field.sortable?'0':null"
                 >
                   <slot :name="'HEAD_'+key" :label="field.label" :column="key" :field="field">
-                    <div v-html="field.label"></div>
+                    <component v-if='field.component' :is='field.component' :field='field' ></component>
```

Comment:
If you do this, your component will be very limited (as in impossible) for any `props` you may need to set on it.

Just  place your component inside a `<b-table ...><template slot="HEAD_<filedkey>">...</template></b-table>`